pkg/agent/route/route_linux.go

```diff
@@ -115,19 +116,46 @@ func (c *Client) Initialize(nodeConfig *config.NodeConfig, done func()) error {
 func (c *Client) initIPTablesOnce(done func()) {
 	defer done()
 	backoffTime := 2 * time.Second
+	stopCh := signals.RegisterSignalHandlers()
```

Comment:
We do not call this function multiple times. We call it once in `main()` and then pass the `stopCh` channel around to all components as needed. IMO, calling `RegisterSignalHandlers` here like you are doing makes it hard to unit test this function down the line if needed.

Reply:
Moved start of goroutine to agent.go ([as per feedback](https://github.com/vmware-tanzu/antrea/pull/1751#discussion_r558579670)). So now we have access to the stopCh, hence removed call to RegisterSignalHandlers.